### ISSUE TITLE
Artisan Worktables not working

### ISSUE THREAD

**Issue by Vrath** [CleanroomMC/Cleanroom#246]
### Your CleanroomMC Discord Username

s0undweaver

### Cleanroom Version

0.2.4-alpha

### Java Version

Java 21

### Graphics Card Vendor

AMD iGPU

### Bug Report

[latest.log](https://github.com/user-attachments/files/17875565/latest.log)

Artisan Worktables doesen't work on clean installation of Cleanroom (with all required dependencies). Clean installation works normally, and so does it work on forge, but for some reason it doesn't work on Cleanroom.
This was also true on 0.2.3

### Mod List

Cleanroom (+ dependencies)
Artisan Worktables (+dependencies)

### MMC Instance Notes

-

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I have installed [Fugue](https://github.com/CleanroomMC/Fugue) and it does not fix this issue.
- [X] I am using experimental branch and have installed [Scalar](https://github.com/CleanroomMC/Scalar/releases/tag/2.11.1) and it does not fix this issue.
- [x] I am running a test build from Cleanroom Github Actions.  (Or, if I've compiled it myself I plan to fix the issue)

**Comment by kappa-maintainer** [comment]
Looks like a gson failure, could you provide the json here? (since it boots normally in my side)

**Comment by Vrath** [comment]
How do I find it?

**Comment by kappa-maintainer** [comment]
Yes, I need the reproduce steps

**Comment by Vrath** [comment]
1) Install cleanroom and dependencies
2) Install artisan worktables and its dependencies

**Comment by kappa-maintainer** [comment]
That's exactly what I did, and I entered the game without any crashes

**Comment by Vrath** [comment]
Earlier you have said that this could be a gson failure and asked for a json, could you elaborate on that?

**Comment by kappa-maintainer** [comment]
The `data` object convert from an unknown json was null, so the game crash
But it seems the mod doesn't come with json by default and I can't reproduce the crash, so there must be something different in your instance

**Comment by KorewaLidesu** [comment]
Can you delete the config folder and try again?

**Comment by Vrath** [comment]
Somehow, perhaps by slightly altering the order of installed jars and launching MC in between (or maybe I also tried to install the addons I wanted back then), I got it to work, but it still wouldn't work on my main instance. Then I compared the config files for both instances and noticed that one of the json files in configs was 0kB, while the fresh one was 8kB. It's strange, cause I'm 100% sure I never touched that folder, but deleting the folder and relaunching the instance to regenerate that file helped. Thank you for help!
